### PR TITLE
PLAT-139057: Update CheckboxItem

### PR DESCRIPTION
**Pull request by stanca-pop-lgp**
Enact-DCO-1.0-Signed-off-by: Stanca Pop stanca.pop@lgepartner.com

### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [x] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [x] Documentation was verified or is not changed
* [x] UI test was passed or is not needed
* [x] Screenshot test was verified or is not needed
* [ ] This is an API breaking change

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Updated CheckboxItem according to the new design requirements for Silicon skin.  Also fixed some styles in Checkbox.

### Links
[//]: # (Related issues, references)
PLAT-139057

### CODE REVIEW

File: CheckboxItem/CheckboxItem.module.less [hunk 1]
```diff
@@ -29,6 +32,11 @@
 			align-items: center;
 			-webkit-margin-start: 0;
 			-webkit-margin-end: @agate-checkboxitem-icon-margin;
+
+          .checkbox {
+            height: @agate-checkboxitem-checkbox-size;
+            width: @agate-checkboxitem-checkbox-size;
```
Comment (daniel-stoian-lgp) [inline CheckboxItem/CheckboxItem.module.less:38]: I don't think this is needed. Current checkbox size of 24px + 2*border-size(3px), is 30px, like in the GUI

File: CheckboxItem/CheckboxItem.module.less [hunk 2]
```diff
@@ -7,16 +7,19 @@
...
 		&.selected {
 			border-color: @agate-checkboxitem-selected-border-color;
+
 			&::before {
 				content: none; // Overwrite the selected styles for Item
 			}
+
+          .focus({
+            border-color: @agate-checkboxitem-focus-border-color;
+          });
```
Comment (daniel-stoian-lgp) [inline CheckboxItem/CheckboxItem.module.less:22]: I'm not sure this is needed. I think we should keep consistency with Item. I will also ask the designer to correct the GUI to specifically show that the border is white with 40% opacity
image

Comment (stanca-pop-lgp) [reply CheckboxItem/CheckboxItem.module.less:22]: But what about this specific requirement?
image

Comment (daniel-stoian-lgp) [reply CheckboxItem/CheckboxItem.module.less:22]: As I said, I'm not sure. This is not a blocker comment

**Comment by daniel-stoian-lgp** [comment]
The horizontal position of the Checkbox is not the same as in the GUI
image
image

File: CheckboxItem/CheckboxItem.module.less (hunk 2 above)
Comment (adrian-cocoara-lgp) [reply CheckboxItem/CheckboxItem.module.less:22]: I think we should leave this as is until we get an answer for this issue

Comment (adrian-cocoara-lgp) [reply CheckboxItem/CheckboxItem.module.less:22]: also please check indentation

**Review by daniel-stoian-lgp — APPROVED** [review]
LGTM

**Review by jeonghee27 — APPROVED** [review]
LGTM

**Pull request merged**